src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,10 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+	const divBy3 = number % 3 === 0;
+	const divBy5 = number % 5 === 0;
+	if (divBy3 && divBy5) return 'FizzBuzz';
+	if (divBy3) return 'Fizz';
+	if (divBy5) return 'Buzz';
+	return number;
```

Comment:
good logic, but you should space this out more to improve readability. There is set way to space out your code, but generally you should try to leave gaps between different jobs/things your trying to do, e.g.
```
const fizzBuzz = (number) => {
  const divBy3 = number % 3 === 0;
  const divBy5 = number % 5 === 0;

  if (divBy3 && divBy5) return 'FizzBuzz';

  if (divBy3) return 'Fizz';

  if (divBy5) return 'Buzz';

  return number;
};
```